backend/src/index.ts

```diff
@@ -403,21 +403,25 @@ app.get("/user/societies", async (req, res: Response) => {
 
   const userID = sessionFromDB.userId;
 
-  const societies_joined = await prisma.society.findMany({
+  const societies = await prisma.society.findMany({

```

Comment:
is there some way we could like process this so that it returns a list of societies you admin and a list of societies you're in? maybe two queries is the way to go. idk.

Reply:
So sorry, didn't see this msg. I've since separated the societies joined and societies administered.